test/dart_cli_test.dart

```diff
@@ -19,6 +21,34 @@ void main() {
     sass.stdout.expect(matches(new RegExp(r"^\d+\.\d+\.\d+")));
     sass.shouldExit(0);
   });
+
+  group("supports package imports", () {
+    test("is found", () {
+      d.file("test.scss", "a {b: 1 + 2}").create('lib');
```

Comment:
Don't create files outside of the sandbox during tests. To properly test this, you'll probably need to add that `--packages` flag.